### PR TITLE
spruce up jseval

### PR DESCRIPTION
**Pull request by albertjan**
@prescod have a play with this see if this works for you.

`jseval("5")` will return `5`

and

`jseval("window.alert('test')")` also still works and returns `None`

and 

`jseval("throw new Error('testing!');")` also still "works"

### CODE REVIEW

File: src/builtin.js [hunk 1]
```diff
@@ -862,7 +862,16 @@ Sk.builtin.raw_input = function (prompt) {
 Sk.builtin.input = Sk.builtin.raw_input;
 
 Sk.builtin.jseval = function jseval (evalcode) {
-    goog.global["eval"](Sk.ffi.remapToJs(evalcode));
+    var result = goog.global["eval"](Sk.ffi.remapToJs(evalcode));
+    try {
+        return Sk.ffi.remapToPy(result);
+    } catch (err) {
+        if (err.constructor === goog.asserts.AssertionError) {
```
Comment (j-f1) [inline src/builtin.js]: Ho come this is special-cased?

**Comment by albertjan** [comment]
If you return something that Sk.ffi.remapToPy can’t remap it throws an AssertionError.

________________________________

**Comment by prescod** [comment]
It seems to work for me, but I guess my deeper question is why are we adding things to the global namespace at all? What's wrong with "from skulpt import jseval" or something like that?

Or perhaps at least call it __jseval__ or something?

Cramming stuff into the main namespace seems a bit unPythonic to me, TBH.

**Comment by prescod** [comment]
Maybe it's a separate issue that I'm complaining about now, though. JSEval definitely works better than it did. Thanks.

**Comment by albertjan** [comment]
@prescod why it is where it is predates my involvement, but changing it now constitutes a breaking change and it doesn't bother me much it's there 😄

**Comment by albertjan** [comment]
@skulpt/committers do we like this? Or are we going to do a separate module so we can `import js`, although I don't know when we'll get to that.

**Comment by meredydd** [comment]
I don't like it in the global namespace either, but given that it is there, making it less likely to explode is an easy accept. Approved.

**Pull request merged**